tests/bluechi_test/bluechictl.py

```diff
@@ -134,6 +134,45 @@ def reload_unit(
             expected_result,
         )
 
+    def reset_failed_node(
+        self,
+        node_name: str,
+        check_result: bool = True,
+        expected_result: int = 0,
+    ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+        return self._run(
+            f"ResetFailed on node {node_name}",
+            f"reset-failed {node_name}",
+            check_result,
+            expected_result,
+        )
+
+    def reset_failed_all_nodes(
+        self,
+        check_result: bool = True,
+        expected_result: int = 0,
+    ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+        return self._run(
+            "ResetFailed on all nodes",
+            "reset-failed",
+            check_result,
+            expected_result,
+        )
+
+    def reset_failed_units(
+        self,
+        node_name: str,
+        unit: str,
+        check_result: bool = True,
+        expected_result: int = 0,
+    ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+        return self._run(
+            f"ResetFailedUnit on node {node_name} on units {unit}",
+            f"reset-failed {node_name} {unit}",
+            check_result,
+            expected_result,
+        )
```

Comment:
Lets unify these three functions. The log message (first param in `_run`) doesn't need to be tailored to the use case and can be a bit general. 

```suggestion
    def reset_failed(
        self,
        node_name: str = "",
        units: List[str] = [],
        check_result: bool = True,
        expected_result: int = 0,
    ) -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
        cmd = f"reset-failed {node} {','.join(units)}".strip()
        return self._run(
            f"ResetFailed on node {node_name} for units {units}",
            cmd,
            check_result,
            expected_result,
        )
```